### DISCUSSION TITLE
Account Suspension

### DISCUSSION THREAD

**Discussion by Cloudzyyyyy** [community/community#140880] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question
Account suspended 

### Body

hey guys, i am Cloudzu, month ago my main github account was hacked and someone spammed suspicious links with it and after that my github account was suspended. i created the ticket with the details of it and they give me link to verify my account and all and said we'll look into the situation. it's been more than month and they are not responding i also created the 2nd ticket about the same and still no reply from their side. i had my repositories saved in my account and had no another backup of it i dont know what to do now..

**Comment by ebndev** [comment, marked as answer]
Hi @Cloudzyyyyy,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
